playground/nextjs-app-router/components/form/component-mode.tsx

```diff
@@ -0,0 +1,34 @@
+import { Label } from '@/components/ui/label';
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from '@/components/ui/select';
+import { useContext } from 'react';
+import { AppContext } from '../AppProvider';
+import type { ComponentMode as ComponentModeReact } from '../AppProvider';
+
+export function ComponentMode() {
+  const { componentMode, setComponentMode } = useContext(AppContext);
+
+  return (
+    <div className="grid gap-2">
+      <Label htmlFor="mode">Component Mode</Label>
+      <Select
+        value={componentMode}
+        onValueChange={(value: ComponentModeReact) => setComponentMode(value)}
+      >
+        <SelectTrigger id="mode">
+          <SelectValue placeholder="Select mode" />
+        </SelectTrigger>
+        <SelectContent>
+          <SelectItem value="auto">Auto</SelectItem>
+          <SelectItem value="light">Light</SelectItem>
+          <SelectItem value="dark">Dark</SelectItem>
+        </SelectContent>
+      </Select>
+    </div>
+  );
+}
```

Comment:
should these files be `ComponentMode.tsx` and `ComponentTheme.tsx` to keep consistent with others or is there a reason you named them differently?

Reply:
I'm not sure why, but we have been using snake-case in this `form` file. So this format matches that.
<img width="339" alt="Screenshot 2024-10-10 at 2 19 18 PM" src="https://github.com/user-attachments/assets/a6eaf96a-0ca5-42e0-945a-6d39ae11f6ce">